packages/grafana-ui/src/utils/debug.ts

```diff
@@ -0,0 +1,30 @@
+import { Logger } from './logger';
+
+/**
+ * Allows debug helpers attachement to the window object
+ * @internal
+ */
+export function attachDebbuger(key: string, thebugger?: any, logger?: Logger) {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
nit: maybe consider a quick exit here instead?
```suggestion
  if (process.env.NODE_ENV === 'production') {
    return;
  }
```